### PR TITLE
Dynamic generators integration

### PR DESCRIPTION
**Pull request by jvlyndark**
temporary fix to the random seed bug included, but otherwise in good working order.

### CODE REVIEW

**Comment by aalex** [comment]
This branch has conflicts that should be resolved. DId you rebase it onto dev?

**Comment by netherwaves** [comment]
sorry, that's my fault! i was planning to hold off on merging the save/load PR because i knew there were going to be conflicts w/ this branch

File: autonomx/WolframCA.cpp [hunk 1]
```diff
@@ -34,111 +34,139 @@ WolframCA::WolframCA(int id, GeneratorMeta * meta) : Generator(id, meta){
...
+                   seedChosen = true;
+               }
+               else {
+                   cells[i] = 0;
+               }
+           }
+       }
+
+       // TODO:
+       // fix this random seed bug
```
Comment (aalex) [inline autonomx/WolframCA.cpp:94]: Which random seed bug?

Comment (jvlyndark) [reply autonomx/WolframCA.cpp:94]: We discussed it in the sprint review. It's working with a temporary fix but not as intended.

**Review by netherwaves — APPROVED** [review]
LGTM, rebase & merge would be the way to go for this one! :)

**Pull request merged**